@ahugeat ,Please take a look:
````
/mnt/vcpkg-ci/buildtrees/gamedev-framework/src/v0.20.0-2c30174aeb.clean/include/gf/Path.h:24:10: fatal error: filesystem: No such file or directory
 #include <filesystem>
          ^~~~~~~~~~~~
compilation terminated.
````

--- a/ports/gamedev-framework/portfile.cmake
+++ b/ports/gamedev-framework/portfile.cmake
@@ -1,3 +1,7 @@
+if (VCPKG_HOST_IS_LINUX)
+    message(WARNING "gamedev-framework requires gcc version 8.3 or later.")
+endif()
+
 vcpkg_fail_port_install(
     ON_ARCH "arm"
 )

--- a/ports/gamedev-framework/vcpkg.json
+++ b/ports/gamedev-framework/vcpkg.json
@@ -1,6 +1,6 @@
 {
   "name": "gamedev-framework",
-  "version-string": "0.20.0",
+  "version-semver": "0.20.0",
   "maintainers": [
     "Julien Bernard <julien.bernard@univ-fcomte.fr>",
     "Arthur Hugeat <hugeat.arthur@gmail.com>"

--- a/scripts/ci.baseline.txt
+++ b/scripts/ci.baseline.txt
@@ -355,6 +355,7 @@ fuzzylite:x64-uwp=fail
 gainput:arm-uwp=fail
 gainput:x64-linux=fail
 gainput:x64-uwp=fail
+gamedev-framework:x64-linux=skip
 gasol:arm64-windows=fail
 gasol:arm-uwp=fail
 gasol:x64-uwp=fail

--- a/versions/g-/gamedev-framework.json
+++ b/versions/g-/gamedev-framework.json
@@ -2,7 +2,7 @@
   "versions": [
     {
       "git-tree": "d3b697d91383d8feef4aadfa4f2f23cab78407e8",
-      "version-string": "0.20.0",
+      "version-semver": "0.20.0",
       "port-version": 0
     },
     {

--- a/versions/g-/gamedev-framework.json
+++ b/versions/g-/gamedev-framework.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "d3b697d91383d8feef4aadfa4f2f23cab78407e8",
+      "git-tree": "461c5eb4f737341e4b800699d1b7021f8bd46c2d",
       "version-semver": "0.20.0",
       "port-version": 0
     },

